darts/models/forecasting/prophet_model.py

```diff
@@ -124,6 +130,29 @@ def __init__(
             Some optional keyword arguments for Prophet.
             For information about the parameters see:
             `The Prophet source code <https://github.com/facebook/prophet/blob/master/python/prophet/forecaster.py>`_.
+
+        Examples
+        --------
+        >>> from darts.datasets import AirPassengersDataset
```

Comment:
P1, P3
also, prophet uses a daily monthly and yearly seasonality by deafult, maybe not the best example to add this seasonality

Reply:
Changed to quarterly, and I added the country holidays in the example.